src/core/Utility.h

```diff
@@ -82,6 +82,9 @@ namespace utility
 			}
 			
 			constexpr auto TICKETS = ":tickets:";
+			constexpr auto STAR = ":star:";
+			constexpr auto LAPTOP = ":computer:";
+			constexpr auto DESKTOP = ":desktop_computer:";
```

Comment:
Can you make the emoji name and the variable name same, to avoid confusion and make it easy to use? 😄 
```cpp
constexpr auto COMPUTER= ":computer:";
constexpr auto DESKTOP_COMPUTER = ":desktop_computer:";
```